README.md

```diff
@@ -166,6 +166,15 @@ You can run every example with zero setup as an "ANONYMOUS" Neptune user (no reg
 | ----------- | :---: | :---: | :------: | :---:
 | GitHub Actions | [![docs]](https://docs.neptune.ai/integrations-and-supported-tools/continuos-integration-and-delivery-ci-cd/github-actions) | | | |
 
+### AWS SageMaker support
+
+| | Docs | Neptune | GitHub |
+| ----------- | :---: | :---: | :------: |
+| Using Neptune in training jobs with custom Docker containers | | [![neptune]](https://app.neptune.ai/common/showroom/e/SHOW-29006) | [![github]](integrations-and-supported-tools/sagemaker/custom-docker-container/) |
```

Comment:
@twolodzko - the tracked source file isn't being rendered in the UI (probably because of the missing `.py` extension).
I know you wanted to keep it as close to the official AWS guide as possible, but I think it'd be better from a user perspective to include the extension so that the source code can be viewed on the app.

Reply:
It's not about being close to the AWS tutorial but that SageMaker expects the `train` executable file to be there. We could add a workaround such that there is a bash file `train` calling `train.py` and passing the arguments to it, but it seems to be an overkill. Also, the syntax highlighting doesn't work because be detect file type only based on extension, while we could as well use the `#!/usr/bin/env` headlines so it works with cases like this.

![image](https://user-images.githubusercontent.com/3750140/210572860-efcb32cf-a5e4-4ff5-987d-5f7b50fba81f.png)